### DISCUSSION TITLE
Feature request: optionally have a start date for Flow streams

### DISCUSSION THREAD

**Discussion by maxdesalle** [sablier-labs/flow#347] [Ideas]
![CleanShot 2024-12-12 at 10 25 37@2x](https://github.com/user-attachments/assets/874a70b5-b5d2-4c7c-bc3c-4d18cb8c35f6)

**Comment by smol-ninja** [comment]
We have created an issue to implement this feature: https://github.com/sablier-labs/flow/issues/351
